org.lflang/src/org/lflang/generator/Main.java

```diff
@@ -195,7 +195,8 @@ public static String redAndBold(String s) {
      * @author Marten Lohstroh <marten@berkeley.edu>
      */
     enum CLIOption {
-        COMPILER("c", "target-compiler", true, false, "Target compiler to invoke.", true),
+        COMPILER(null, "target-compiler", true, false, "Target compiler to invoke.", true),
+        CLEAN("c", "clean", false, false, "Clean before building.", true),
```

Comment:
This adds the flag, but `context` is not set in response to option being enabled. Are you sure this even works yet?

Reply:
I don't understand this comment. Yes it works :) Since the last boolean in the constructor (`passOn`) is set to true, some magic elsewhere in the Main class automatically adds the option to the context.

Reply:
Ah, yes. I wrote that magic :-)